app/src/main/java/ar/com/wolox/android/example/di/AppModule.kt

```diff
@@ -43,4 +46,13 @@ abstract class AppModule {
 
     @ContributesAndroidInjector
     internal abstract fun profileFragment(): ProfileFragment
+
+    @ContributesAndroidInjector
```

Comment:
To avoid having so much `Contributes` in one file, we can move them in different Modules, like: `LoginModule`, `NewsModule`, etc.